src/ResourceManagement/Azure.Fluent/Azure.cs

```diff
@@ -320,6 +322,15 @@ public IDnsZones DnsZones
             }
         }
 
+        /// <returns>entry point to managing private DNS zones</returns>
```

Comment:
managing ==> manage

Reply:
Should we replace all? The `managing` is everywhere in this file.

Reply:
Changed to all.